README.md

```diff
@@ -1,2 +1,28 @@
 # agent-packaging
 Combined packaging scripts for the Stackdriver monitoring agent
+
+## Branches in Use
+
+* master
+
+  We use this branch to build the 6.X.X agent DEB and RPM packages for the Cloud Rapture repos.
+  
+* stackdriver-agent-5.5.2
+
+  We use this branch to build the 5.X.X agent DEB and RPM packages for the Cloud Rapture repos.
```

Comment:
[Optional] `5.5.2-X agent DEB and RPM packages`...

Reply:
Done